java/kotlin-extractor/src/main/kotlin/KotlinFileExtractor.kt

```diff
@@ -404,8 +404,7 @@ open class KotlinFileExtractor(
                 }
 
                 extractClassModifiers(c, id)
-                val forceExtractSupertypeMembers = !isExternalDeclaration(c)
-                extractClassSupertypes(c, id, inReceiverContext = forceExtractSupertypeMembers)
+                extractClassSupertypes(c, id, inReceiverContext = true)
```

Comment:
Would it be worth adding a comment giving the rationale for `inReceiverContext = true` here?